package.json

```diff
@@ -52,5 +54,8 @@
 		"node": ">= 12.22.1",
 		"npm": ">= 6.13.4",
 		"yarn": ">= 1.21.1"
+	},
+	"jscpd": {
+    "threshold": 5
```

Comment:
This needs to go in the .jscpd.json file (added in my pull request).  jscpd honors the block in the package.json file, but I don't think that super-linter does.  Furthermore, I would rather not clutter the package.json file.

Reply:
added this as well.